### ISSUE TITLE
Gibberish in chrysanthemumgarden.com

### ISSUE THREAD

**Issue by bookimp** [dteviot/WebToEpub#967]
**Describe the bug**
epubs from chrysanthemumgarden.com have gibberish text at the end of many lines as well as sentences full of agregator blocker gibberish. Also large bold font apologizing for it. 

**To Reproduce**
1. Go to https://chrysanthemumgarden.com/novel-tl/nhsre/
2. Download epub with default settings
3. Scroll down to the 1st or 2nd page of epub, should see gibberish at the end of some paragraphs
4. It would be nice to also get rid of 'Please support our translators' but I can live with those. 

**Expected behavior**
No gibberish that wasn't in the original.

**Screenshots**
![image](https://user-images.githubusercontent.com/20101896/233274064-aed1d676-838a-4854-81a5-ad0e09fd12ba.png)

**Desktop (please complete the following information):**
OS: Windows 10 64bit
Browser: Opera GX
Version LVL4 (core: 97.0.4719.84)

**Comment by dteviot** [comment]
@bookimp 

Apologies for delay in responding, I've had RL issues for the last 4 weeks.
Anyway, your problem is a duplicate of https://github.com/dteviot/WebToEpub/issues/354

Short answer:

1. Go to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing and download EpubEditor.zip
2. Unpack the zip and open the file main.html in chrome.
3. Drag and drop Chrysanthemum Garden epub onto the drop zone.
4. A set of buttons will appear, click the second from bottom labelled "Clean ChrysanthemumGarden"
5. Wait a little bit. You should get a file dialog asking you for name to save the fixed epub.
6. 

Slightly longer answer.
Basically, in order to prevent the chapters on site being copied, the site scrambles the text and uses a special font to unscramble the text. Because this is done as a copyright mechanism, if WebToEpub was to unscramble the text it would get WebToEpub removed from the chrome store.
However, I have provided a tool to unscramble the file that's not on the Chrome store.
